cpp/arcticdb/storage/s3/s3_storage.hpp

```diff
@@ -245,7 +245,9 @@ auto get_s3_config(const ConfigType& conf) {
     auto endpoint = conf.endpoint();
     util::check_arg(!endpoint.empty(), "S3 Endpoint must be specified");
     client_configuration.endpointOverride = endpoint;
-    client_configuration.verifySSL = false;
+    const bool use_ssl = ConfigsMap::instance()->get_int("S3Storage.UseSSL", conf.ssl());
```

Comment:
`VerifySSL` not `UseSSL`.

SSL will be used - we're just not verifying the certificate. Different things.